src/Textbox/Textbox.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Textbox from './Textbox';
+import renderer from 'react-test-renderer';
+import { mount, shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+describe('<Textbox />', () => {
+  describe('behaviour tests', () => {
+    let text = '';
+    let component;
+    let tree;
+    let reRender;
+    const makeRenderer = component =>
+      () => tree = component.toJSON();
+
+    beforeEach(() => {
+      component = renderer.create(
+        <Textbox
+          onChange={(event) => {text = event} }
+          value={text} />
+      );
+      reRender = makeRenderer(component);
+      reRender();
+    });
+
+    it('match snapshot', () => {
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('write text in the input', () => {
```

Comment:
To express better what you are doing in this test, change the test description to `registers onChange function into the component tree`